### ISSUE TITLE
Extract OCR to current file

### ISSUE THREAD

**Issue by kaiogu** [schlundd/obsidian-ocr-plugin#4]
Hello,

would it be possible create an alternative command for the plugin, `Taskbone OCR: Extract text into current file for image` to extract the Text into the current file instead of into a separate file?

Thanks for the great plugin!

**Comment by schlundd** [comment]
Ok, it took a while... This is finally possible, by using the new Taskbone Plugin www.taskbone.com. Check out https://docs.taskbone.com/updates/2022-05-27-ocr-and-new-output-options. This is probably exactly what you meant?

![Peek 2022-05-27 16-53](https://user-images.githubusercontent.com/1160643/170772341-775ee2de-2473-4dc8-bded-a11d0deb7220.gif)


The feature will be backported into this plugin, but there is no timeline.

**Comment by DryIce1** [comment]
Hi @schlundd,

I just registered for an API and early access.  I left a note with my use case in the sign up form, but I will repeat it here, just in case it is popular.

**current problem**: I have pasted many images into my vault (mostly question/answer, web page screenshot). Now my vault size is too large, and obsidian opening time on my mobile device is too long. So as I am revising, I would like to go through my notes and OCR the images. 

**possible solutions**:
1. OCR image under cursor and output text on the next line
2. OCR all images in the current markdown file, and output the text immediately after each image.

I don't think I was able to set up these commands in the current version of taskbone 1.2

thanks
